BD-2982 Update Shopify back-in-stock notifications article (#7174)
braze-inc/braze-docs: _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_features/shopify_catalogs/back_in_stock.md

Back-in-stock notifications for Braze Shopify catalogs are currently in early access. Contact your account manager if you're interested in participating in this early access.
{% endalert %}

> When a customer clicks an item, we'll automatically subscribe them to receive back-in-stock notifications for that item. Once the item's inventory quantity is greater than zero, all subscribers will be elgible for notifications through a campaign or Canvas. However, only users who opted into notifications will receive notifications. 

Reviewer: ```suggestion
When a customer clicks an item, we'll automatically subscribe them to receive back-in-stock notifications for that item. Once the item's inventory quantity is greater than zero, all subscribers will be eligible for notifications through a campaign or Canvas. However, only users who opted into notifications will receive notifications. 
```

{% raw %}
To template in details about the catalog item that's back in stock, you can use the `canvas_entry_properties` Liquid tag to access the `item_id`. 

To return the ID of the item that is back in stock, use the Liquid tag ``{{canvas_entry_properties.${catalog_update}}}``. To access data about that item throughout your message, put the Liquid tag  ``{% catalog_items <name_of_your_catalog> {{canvas_entry_properties.${catalog_update}}} %}`` at the top of your message, then use ``items[0].<field_name>` to place the data in your message.
{% endraw %}

Here is a sample Liquid template that will display the item's product title, image URL, and price:

{% raw %}
```liquid
{% catalog_items shop-1234_shopify_catalog {{canvas_entry_properties.${catalog_update}.item_id}} %}
{{ items[0].shopify_product_title }}
{{ items[0].product_image_url }}
{{ items[0].price }}
```
{% endraw %}

## Considerations

> - Users are only subscribed for 90 days. If the item isn't back in stock in 90 days, the user is unsubscribed.
> - When using the **Notify all subscribed users** notification rule, Braze will notify 100,000 over 10 minutes.
> - Braze will process at most 10 item updates over one minute. If you update 11 items in one minute, only the first 10 will be able to trigger a back-in-stock notification.

Reviewer: ```suggestion
- Users are only subscribed for 90 days. If the item isn't back in stock in 90 days, the user is unsubscribed.
- When using the **Notify all subscribed users** notification rule, Braze will notify 100,000 over 10 minutes.
- Braze will process at most 10 item updates over one minute. If you update 11 items in one minute, only the first 10 can trigger a back-in-stock notification.
```
